decktape.js

```diff
@@ -244,6 +244,16 @@ process.on('unhandledRejection', error => {
 
 })();
 
+async function writePdf(filename, pdf) {
+  const pdfDir = path.dirname(filename);
+  try {
+    fs.accessSync(pdfDir, fs.constants.F_OK);
+  } catch {
+    fs.mkdirSync(pdfDir);
```

Comment:
Would it make sense to use `recursive: true`? It’s been added in Node 10.12.0, which is already a requirement for Puppeteer 3+. See https://nodejs.org/api/fs.html#fs_fs_mkdirsync_path_options.

Reply:
It would totally made sense 👍  I add the option 😄

Reply:
Done 👍  Thanks for the review!